src/tests/eolian/eolian_static.c

```diff
@@ -10,6 +10,7 @@ EFL_START_TEST(eolian_static_check)
 {
    Eolian_State *eos = eolian_state_new();
    fail_if(!eolian_state_directory_add(eos, EO_SRC_DIR));
+   fail_if(!eolian_state_directory_add(eos, EO_BUILD_DIR));
```

Comment:
Why do we need this? (And what does this test-case tests exactly?)

Reply:
This kinda loads all the `.eo`s at that directory, the next line compiles them all. As now there are some `eo`s classes at the build dir, we need to include them as well to address all dependencies.